src/core/Akka.Persistence.Tests/PersistentActorSpecAsyncAwait.Actors.cs

```diff
@@ -0,0 +1,1337 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="PersistentActorSpec.Actors.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using Akka.Actor;
+using Akka.TestKit;
+using Akka.Util.Internal;
+using Akka.Persistence.Internal;
+using System.Threading.Tasks;
+
+namespace Akka.Persistence.Tests
+{
+    //internal class BehaviorOneActor : PersistentActorSpec.ExamplePersistentActor
```

Comment:
I had to move it to another namespace. It inherits from PersistentActorSpec.ExamplePersistentActor but i need to use a different actor with the RunTask inside